Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM metabase/metabase:latest
+ADD https://github.com/MaterializeInc/metabase-materialize-driver/releases/download/0.0.1/materialize-driver-0.0.1-SNAPSHOT-standalone.jar /plugins/
+RUN chmod 744 /plugins/materialize-driver-0.0.1-SNAPSHOT-standalone.jar
```

Comment:
I think technically this should include a trailing newline. Stylistically I'd definitely prefer it if it did.